### PR TITLE
added tests for environment, flickr_get_photos(), flickr_get_photo()

### PR DESCRIPTION
**Pull request by decobots**

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import unittest
+from environment_variables import TWITTER_CONSUMER_KEY, TWITTER_CONSUMER_SECRET, TWITTER_ACCESS_KEY, \
+    TWITTER_ACCESS_SECRET, FLICKR_API_KEY
+from main import flickr_get_photos_list, flickr_get_photo
+
+
+class TestEnvironmentVariables(unittest.TestCase):
```
Comment (livercat) [inline tests.py]: You don't really need these tests. os.getenv() always returns strings, and checking for non-empty values should be done in your app, not in tests.

File: tests.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    def test_environment_variables_FLICKR_API_KEY(self):
+        self.assertIsInstance(FLICKR_API_KEY, str)
+        self.assertFalse(FLICKR_API_KEY == '')
+
+
+class TestFlickr(unittest.TestCase):
+    def test_flickr_get_photos_list_correct(self):
+        """
+        check that returned value is list and attribute farm, server, id, secret exists for each list item
```
Comment (livercat) [inline tests.py]: "attribute ... exists" -> "attributes ... exist"

File: tests.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        for photo in flickr_get_photos_list_result:
+            self.assertIsInstance(photo, dict)
+            self.assertIsNotNone(photo["farm"])
+            self.assertIsNotNone(photo["server"])
+            self.assertIsNotNone(photo["id"])
+            self.assertIsNotNone(photo["secret"])
+
+    def test_flickr_get_photo_correct(self):
+        """
+        check that returned value are exist and have types bytes and string
```
Comment (livercat) [inline tests.py]: "are" is not needed here

File: tests.py (hunk 1 above)
Comment (decobots) [reply tests.py]: Changed

File: tests.py (hunk 3 above)
Comment (decobots) [reply tests.py]: fixed, thank you!

**Pull request merged**